sites/NOAA_GFDL/default_gfdl.jsonc

```diff
@@ -22,21 +22,21 @@
       "CASE_ROOT_DIR" : "/archive/oar.gfdl.cmip6/ESM4/DECK/ESM4_historical_D1/gfdl.ncrc4-intel16-prod-openmp/pp",
       "pod_list": [
           // Optional: PODs to run for this model only (defaults to all)
-          "Wheeler_Kiladis",
-          "EOF_500hPa",
-          "convective_transition_diag",
-          "MJO_suite",
-          "MJO_teleconnection",
-          "MJO_prop_amp",
-          "precip_diurnal_cycle",
-          "SM_ET_coupling"
+          //"Wheeler_Kiladis",
+          "EOF_500hPa"
+         // "convective_transition_diag",
+         // "MJO_suite",
+         // "MJO_teleconnection",
+         // "MJO_prop_amp",
+         // "precip_diurnal_cycle",
+        //  "SM_ET_coupling"
```

Comment:
Not sure I follow why the PODs are disabled at GFDL?  Maybe leftover from testing?